overlord/hookstate/ctlcmd/helpers.go

```diff
@@ -69,6 +69,18 @@ func getServiceInfos(st *state.State, snapName string, serviceNames []string) ([
 
 var servicechangeImpl = servicestate.Change
```

Comment:
Can we please rename this to servicestateChange (or *Control)?  That's the usual name we give to those variables.

Reply:
Ok, fixed in the prerequisite #4065 .